sophos_central_cloud/README.md

```diff
@@ -1,42 +1,60 @@
-# Agent Check: Sophos Central Cloud
+# Sophos Central Cloud Integration for Datadog
 
 ## Overview
 
-This check monitors [Sophos Central Cloud][1].
+[Sophos Central][1] is a unified cloud-based management platform provided by Sophos, a company specializing in digital security. It allows users to manage various Sophos security products from a single interface. It's designed to simplify security management for businesses of all sizes by consolidating multiple security products into a single, cloud-based platform.
 
-## Setup
+This integration ingests the following logs:
+
+- Alerts
+- Events
```

Comment:
please include a brief description that distinguishes what is found in this API.

Reply:
Yes, We have added a brief description.